plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -361,6 +363,8 @@ public class HiveMetadata
     private static final String CSV_QUOTE_KEY = "quoteChar";
     private static final String CSV_ESCAPE_KEY = "escapeChar";
 
+    public static final String PARQUET_BLOOM_FILTER_COLUMNS_KEY_PREFIX = "parquet.bloom-filter-enabled.column.";
```

Comment:
Change this to `parquet.bloom.filter.columns`. It should be comma separated list of columns